js/blocks/components/server-side-render.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Forked from Gutenberg, with a minor edit to allow using POST requests instead of GET requests.
+ * Todo: delete if this is merged: https://github.com/WordPress/gutenberg/pull/21068/
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/c72030189017c8aac44453c1386f4251e45e80df/packages/server-side-render/src/index.js
```

Comment:
This is copied almost verbatim from Gutenberg. 

[index.js](https://github.com/WordPress/gutenberg/blob/c72030189017c8aac44453c1386f4251e45e80df/packages/server-side-render/src/index.js) and [server-side-render.js](https://github.com/WordPress/gutenberg/blob/c72030189017c8aac44453c1386f4251e45e80df/packages/server-side-render/src/server-side-render.js) are copied into this file.